PraceDomowe/PracaDomowa5/Michał Przerwa PracaDomowa5/Michał_Przerwa_PracaDomowa5.ipynb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
<p>To nie jest dobrze, w ogóle nie należało w tym zadaniu używać tabelki <code>comments</code>. Jednak poza tym pomysł dobry. </p><p> </p><p><code>0.5 punktu</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>